extensions/datavirtualization/config.json

```diff
@@ -0,0 +1,24 @@
+{
+	"downloadUrl": "https://sqlopsextensions.blob.core.windows.net/extensions/datavirtualization/service/{#version#}/scaleoutdataservice-{#fileName#}",
+	"version": "1.10.0",
+	"downloadFileNames": {
+		"Windows_64": "win-x64.zip",
+		"Windows_86": "win-x86.zip",
+		"OSX": "osx-x64.tar.gz",
+		"Linux_64": "linux-x64.tar.gz",
+		"CentOS_7": "linux-x64.tar.gz",
+		"Debian_8": "linux-x64.tar.gz",
+		"Fedora_23": "linux-x64.tar.gz",
+		"OpenSUSE_13_2": "linux-x64.tar.gz",
```

Comment:
Didn't know that these platforms were supported.I should add this to import extension as well.

Reply:
You don't need to - the linux config in the import extension will cover all the various Linux distros. I plan on changing this to something similar as one of my follow up tasks.

Reply:
@aasimkhan30 remember that I've explained to you about this? you only need to specify a specific configuration it needs to use a different build. :)